contracts/CollectiveGovernance.sol

```diff
@@ -86,16 +92,29 @@ contract CollectiveGovernance is Governance, VoteStrategy, ERC165 {
     /// @param _supervisorList the list of supervisors for this project
     /// @param _class the VoterClass for this project
     /// @param _governanceStorage The storage contract for this governance
+    /// @param _name The project name
+    /// @param _url The Url for this project
+    /// @param _description The project description
     constructor(
         address[] memory _supervisorList,
         VoterClass _class,
-        Storage _governanceStorage
+        Storage _governanceStorage,
+        bytes32 _name,
+        string memory _url,
+        string memory _description
     ) {
+        require(_supervisorList.length > 0, "Supervisor required");
```

Comment:
I think that’s dangerous though because then no one will be able to edit anything. I think upon creation supervisor should be required but it can be removed down the line once everything is set up for that community

Reply:
I like the change you are suggesting but I do worry that it could have some unintended consequences.   I think we should carve off some time in our meeting today and discuss this.   I would consider the above with regard to a couple different frameworks:

1.  is this change consistent with the original goals we set out for this contract
2. does this change move us closer to being able to release the contract 
3. does the change accomplish the goal of being trustless without unintended side effects

For these my answers would be no, no and maybe, which gives me pause.    Still, I think we should write more and think more on possible trustless models and we might want to introduce another contract that follows that kind of approach in the near future.